cli/cmd/testdata/install_controlplane_tracing_output.golden

```diff
@@ -1261,9 +1263,9 @@ spec:
         - --proxy-uid
         - "2102"
         - --inbound-ports-to-ignore
-        - "4190,4191,25,443,587,3306,11211"
+        - "4190,4191"
         - --outbound-ports-to-ignore
-        - "25,443,587,3306,11211"
+        - ",443"
```

Comment:
The control plane deployments templates assume there is more than one port here so they prepend that unwanted comma:
```
initContainers:
       {{- if not (contains "443" ( .Values.proxyInit.ignoreOutboundPorts | toString ))}}
       {{- $_ := set $tree.Values.proxyInit "ignoreOutboundPorts" (printf "%s,443" ( .Values.proxyInit.ignoreOutboundPorts | toString )) -}}
       {{- end}}
```

Reply:
could we do something like this?

```
{{- $outboundPorts := split "," .Values.proxyInit.ignoreOutboundPorts -}}
{{- $_ := set $tree.Values.proxyInit "ignoreOutboundPorts" (append $outboundPorts "443" | uniq | join ",") -}}
```

Reply:
Something like that seems like a good way to solve this. There seems to be something wrong with the syntax, but hopefully can keep it close to that suggestion.

Reply:
I tried the following i.e without the syntax errors
```yaml
{{- $outboundPorts := splitList "," .Values.proxyInit.ignoreOutboundPorts -}}
{{- $_ := set $tree.Values.proxyInit "ignoreOutboundPorts" (append $outboundPorts "443" | uniq | join ",")}}
```

but still could not get it to remove `,` weirdly. I could narrow it down to `join` being the issue with it adding `,` when the list is only a single element (which is in our test cases)

We could instead fix this by doing
```yaml
      initContainers:
      {{- if not (contains "443" ( .Values.proxyInit.ignoreOutboundPorts | toString ))}}
      {{- $_ := set $tree.Values.proxyInit "ignoreOutboundPorts" ( ternary "443" (printf "%s,443" (.Values.proxyInit.ignoreOutboundPorts | toString )) (empty .Values.proxyInit.ignoreOutboundPorts)) -}}
      {{- end}}
```
This is def a bit complex but works.